04 DisplayData/src/components/members/membersPage.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import memberEntity from '../../api/memberEntity';
-import MemberAPI from '../../api/memberAPI';
+import memberAPI from '../../api/memberAPI';
```

Comment:
Next step is replace default exports by import { MemberAPI } from ....
If we want import as camelCase we should :

import { MemberAPI as memberAPI } from .... due to MemberAPI class is defined like PascalCase
Which do think is better option? This one? or import as PascalCase?